core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -941,6 +945,7 @@ public Command workerHeartbeat(long workerId, Map<String, Long> capacityBytesOnT
       worker.updateLastUpdatedTimeMs();
 
       List<Long> toRemoveBlocks = worker.getToRemoveBlocks();
+      Metrics.TOTAL_BLOCKS.inc(addedBlocks.size() - removedBlockIds.size());
```

Comment:
Are we accounting for lost workers somewhere?

Reply:
good point, fixed

Reply:
In addition to this, should we do subtract when a worker gets lost?